parser/lex_buffer.re

```diff
@@ -121,28 +125,8 @@ let utf8 = (~skip=0, ~drop=0, lexbuf) => {
 
 let container_lnum_ref = ref(0);
 
-let fix_loc = loc => {
-  let fix_pos = pos => {
-    /* It looks like lex_buffer.ml returns a position with 2 extra
-     * chars for parsed lines after the first one. Bug? */
```

Comment:
this workaround wasn't needed(anymore?) and was introducing a bug when combining with the new workaround for properties. As we now have a better test suite this should be safe to change.